x-pack/legacy/plugins/canvas/public/components/sidebar_header/sidebar_header.tsx

```diff
@@ -349,6 +475,14 @@ export class SidebarHeader extends Component<Props, State> {
       panels.push(panel);
     }
 
+    const menuFiller = ({ menuItem, panel }: MenuTuple) => {
+      items.push(menuItem);
+      panels.push(panel);
+    };
+
+    this._getAlignmentMenuItems(close).forEach(menuFiller);
+    this._getDistributionMenuItems(close).forEach(menuFiller);
```

Comment:
Nit: You implemented this differently than I did with `_getLayerMenuItems`. Why do you return the object wrapped in an array here instead of just the `MenuTuple` object? 

Since `_getLayerMenuItems`, `_getAlignmentMenuItems`, and `_getDistributionMenuItems` are all similar functions, they should be implemented the same way, so it'd be great if you could refactor `_getLayerMenuItems` like the two new functions or vice versa.

Also, would it make sense to take the conditional check for the number of `selectedNodes` out of the individual functions and wrap the condition around these two function calls?

Here's an example of how I imagine this code looking (with all of the functions returning a `MenuTuple` object)
```suggestion
    const menuFiller = ({ menuItem, panel }: MenuTuple) => {
      items.push(menuItem);
      panels.push(panel);
    };
    
    if (showLayerControls) {
      menuFiller(this._getLayerMenuItems();
    }	
    if(selectedNodes.length > 2) {
      menuFiller(this._getAlignmentMenuItems(close));
      menuFiller(this._getDistributionMenuItems(close));
    }
```

Or alternatively (with all of the functions returning a `MenuTuple[]` array):

```suggestion
    const menuFiller = ({ menuItem, panel }: MenuTuple) => {
      items.push(menuItem);
      panels.push(panel);
    };
    
    if (showLayerControls) {
      this._getLayerMenuItems(close).forEach(menuFiller);
    }	
    if(selectedNodes.length > 2) {
      this._getAlignmentMenuItems(close).forEach(menuFiller);
      this._getDistributionMenuItems(close).forEach(menuFiller);
    }
```

Reply:
@cqliu1 thanks for the feedback, looking at it with a fresh pair of eyes, uniformity and just plain simplicity supports your take, now pushed